.gitignore

```diff
@@ -1,2 +1,3 @@
 **/.DS_Store
-**/.vscode
\ No newline at end of file
+**/.vscode
+test.txt
```

Comment:
why is the test.txt still in this PR although it's in the gitignore file?

Reply:
I'm trying to figure that out too. Adding the filename to the .gitignore doesn't seem to change anything for me and I'm not sure why.

Reply:
Marking this as resolved to show that it's been addressed.